### DISCUSSION TITLE
searchTickets issue

### DISCUSSION THREAD

**Discussion by paskuale75** [simialbi/yii2-ticket#10] [General]
hi @simialbi  i noticed that the search of searchTicket.php doesn't work very well, i noticed it when i added a foreign key and i had to filter for it, I updated this code :
```
$this->load($params);

        if (!$this->validate()) {
            return $dataProvider;
        }

```
with this:

```
if (!$this->load($params) && $this->validate()) {
            return $dataProvider;
        }
```

...and work!

**Comment by simialbi** [comment]
Hmmm... Makes no sense like this, like this you don't load the filter values 🤔.

**Comment by paskuale75** [comment]
Can be as you say but I added a foreign key and without this change, the filter was passed in the `$params` but did not filter the data in the grid.
